### PR TITLE
LPS-76015 petra-content

### PR DESCRIPTION
**Pull request by Preston-Crary**
Please also see:
https://github.com/shuyangzhou/com-liferay-dynamic-data-mapping/pull/12
https://github.com/shuyangzhou/com-liferay-journal/pull/56

Once those are merged we should be able to remove the module.

### CODE REVIEW

File: modules/apps/forms-and-workflow/calendar/calendar-service/src/main/java/com/liferay/calendar/notification/impl/NotificationUtil.java [hunk 1]
```diff
@@ -85,7 +85,7 @@ public static String getDefaultTemplate(
 		String templatePath = CalendarServiceConfigurationUtil.get(
 			propertyName, filter);
 
-		return ContentUtil.get(
+		return StringUtil.read(
```
Comment (shuyangzhou) [inline modules/apps/forms-and-workflow/calendar/calendar-service/src/main/java/com/liferay/calendar/notification/impl/NotificationUtil.java]: this should  catch IOException and log it, to preserve the original behavior.

File: modules/apps/foundation/login/login-web/src/main/java/com/liferay/login/web/internal/portlet/action/LoginConfigurationActionImpl.java [hunk 2]
```diff
@@ -87,4 +81,32 @@ public void processAction(
...
+
+	private static final String _ADMIN_EMAIL_PASSWORD_RESET_SUBJECT;
+
+	private static final String _ADMIN_EMAIL_PASSWORD_SENT_BODY;
+
+	private static final String _ADMIN_EMAIL_PASSWORD_SENT_SUBJECT;
+
+	static {
+		try {
+			_ADMIN_EMAIL_PASSWORD_RESET_BODY = StringUtil.read(
```
Comment (shuyangzhou) [inline modules/apps/foundation/login/login-web/src/main/java/com/liferay/login/web/internal/portlet/action/LoginConfigurationActionImpl.java]: Same thing here, these could be missing, we should not fail the class loading.

**Comment by shuyangzhou** [comment]
@Preston-Crary I merged the subrepo ones, but for the portal side usages removal, please make sure the behavior is consistent as before. Unless we know for sure it is illegal to be missing. Please resend, thanks!

File: modules/apps/forms-and-workflow/calendar/calendar-service/src/main/java/com/liferay/calendar/notification/impl/NotificationUtil.java (hunk 1 above)
Comment (Preston-Crary) [reply modules/apps/forms-and-workflow/calendar/calendar-service/src/main/java/com/liferay/calendar/notification/impl/NotificationUtil.java]: Yes, I'll update this change in the next pull.

**Comment by Preston-Crary** [comment]
Please see https://github.com/shuyangzhou/liferay-portal/pull/5303

**Pull request closed without merging**